tests/h/activity/bucketing_test.py

```diff
@@ -179,6 +180,22 @@ def test_eq_title_mismatch(self, document):
 
         assert not bucket_1 == bucket_2
 
+    def test_incontext_link_returns_link_to_first_annotation(self, document, patch):
+        incontext_link = patch('h.links.incontext_link')
+        bucket = bucketing.DocumentBucket(document)
+        ann = factories.Annotation()
+        bucket.append(ann)
+        request = Mock()
+
+        assert bucket.incontext_link(request) == incontext_link.return_value
```

Comment:
Does this test what the method name says it tests? I don't see how it tests that it was the _first_ annotation that was passed to `incontext_link()`